docs/writing-docs/mdx.md

```diff
@@ -254,36 +285,67 @@ import Changelog from '../CHANGELOG.md';
 
 -->
 
-## Linking to other stories and pages
+## Troubleshooting
 
-When writing MDX, you may want to provide links to other stories or documentation pages and sections. You can use the `path` query string.
+### The migration seems flaky and keeps failing
 
-Considering a story with ID `some--id`, this redirects to the **Docs** tab of the story:
+By default, running the [migration command](#automigration) will try and migrate all existing MDX files in your project according to the MDX 2 specification. However, this might not always be possible, and you might run into issues during the migration. To help you troubleshoot those issues, we've prepared some recommendations that might help you.
 
-```md
-[Go to specific documentation page](?path=/docs/some--id)
+Start by running the following command inside your project directory:
+
+```shell
+npx @hipster/mdx2-issue-checker
 ```
 
-This redirects to the **Canvas** tab of the story:
+<div class="aside">
+💡 Depending on the volume, you may be required to run the command multiple times to fix all the issues.
+</div>
+
+When it finishes, it will output the list of files causing issues. You can then use this information to fix the problems manually.
 
-```md
-[Go to specific story canvas](?path=/story/some--id)
+Additionally, if you're working with VSCode, you can add the [MDX extension](https://marketplace.visualstudio.com/items?itemName=unifiedjs.vscode-mdx) and enable MDX experimental support for linting, type checking, and auto completion by adding the following to your user settings:
+
+```json
+{
+  "mdx.experimentalLanguageServer": true
+}
 ```
 
-You can also use anchors to target a specific section of a page:
+If you're still encountering issues, we recommend reaching out to the maintainers using the default communication channels (e.g., [Discord server](https://discord.com/channels/486522875931656193/570426522528382976), [GitHub issues](https://github.com/storybookjs/storybook/issues)).
 
-```md
-[Go to the conclusion of the documentation page](?path=/docs/some--id#conclusion)
-```
+### The MDX documentation doesn't render in my environment
 
-<div class="aside">
-💡 By applying this pattern with the Controls addon, all anchors will be ignored in Canvas based on how Storybook handles URLs to track the args values.
-</div>
+As Storybook relies heavily on MDX 2 to render documentation, there might be some cases where you cannot render it depending on your project setup. If that's the case, we support MDX 1 as a fallback. To enable it, you'll need to take some additional steps.
```

Comment:
I would pitch this more as: "if you don't want to migrate all your files to MDX2 right now..."